README.md

```diff
@@ -19,6 +19,52 @@ additional records with a standardized template.
 - don't commit your .env or .env.development, but do commit .env.test after
   confirming your test values are not actual secrets that need protecting
 
+## Confirming functionality after updating dependencies
+
+This application has good code coverage, so most issues are detected by just running tests normally:
+
+```shell
+bin/rails test
+```
+
+The following additional manual testing should be performed in the PR build on Heroku.
```

Comment:
This structure looks pretty clear to me, and I agree with the recommendations being made here. Thanks for including specific steps to take, and including the same searches to perform - that's very repeatable :-)